ffi-gdal.gemspec

```diff
@@ -18,19 +18,19 @@ Gem::Specification.new do |spec|
   end
 
   spec.require_paths = %w[lib]
-  spec.required_ruby_version = '>= 2.6'
+  spec.required_ruby_version = '>= 3.0'
```

Comment:
Are we ready to do this?  Currently only kraken and twilight are on 3.0

Reply:
IDK, seems like it's worth setting the bar here now. If kraken and twilight can go 3.0, it seems reasonable that other services shouldn't be that far off? If most services are still on 2.6 and that came out 3 years ago, I'd rather push to get us using current versions of things.

Reply:
I'm a little concerned that we'll be forcing services to upgrade both gdal and ruby at the same time.  Just getting gdal upgraded could be a large endeavor for logged data, even without throwing a ruby on top of it.  This also seems a little arbitrary -- why is it up to ffi-gdal to tell other services to upgrade?  Unless ffi-gdal is using ruby 3.0 specific features, we should allow it to work on 2.6.

Reply:
Ok, reverted back to 2.6.